[fix] 매거진 화면 팝업 모드 매거진 사이즈보다 screenWidth가 작아지는 경우 나타나는 크래시 수정 (#52)

2chang5 opened the pull request:
### 📌 관련 이슈

- closed #52

### 📁 작업 설명

- 이슈 참고 부탁드립니다.

2chang5 committed:
[fix] 매거진 화면 사이즈 관련 크래시 대응 (#52)

easyhooon committed:
[feat] 디버그 모드에서 웹 콘텐츠 디버깅 활성화 (#52)

easyhooon commented on presentation/src/main/kotlin/com/nexters/ziine/android/presentation/magazine/MagazineScreen.kt:
```diff
@@ -169,7 +169,10 @@ private object PagerItemShrinker {
         pageItemWidth: Int,
-    ): Int = (screenWidth - pageItemWidth) / 2
+    ): Int {
+        if (screenWidth <= pageItemWidth) return 0
```
👍 👍 👍  이러면 negative padding 에러는 발생안할듯 하네요!

easyhooon reviewed (approved):
#45 까지 함께 작업 완료 후, 문제 없으면 v1.0.3 으로 앱 업데이트 하면 될 것 같습니다! 

수고하셨습니다~

easyhooon commented:
매거진 클릭시 햅틱도 추가해주세요~

easyhooon committed:
[feat] 누락된 햅틱 로직 추가 (#52)

햅틱 호출 로직 ObserveAsEvents 내부로 이동시켜 중앙화

easyhooon committed:
[fix] 매거진 화면 예외 처리 구현 (#52)

API 호출 실패시 ErrorDialog 호출
API 호출 실패시, magazines.size = 0 으로 인한 ArithmeticException(divide by zero) 관련 예외 처리 추가

easyhooon committed:
[refactor] toPx() toDP() 함수 @ReadOnlyComposable 추가 (#52)
